src/website/views/components/toolbar/_examples.pug

```diff
@@ -1304,22 +1618,73 @@ h2 Examples
               <chi-link href="#" size="md">Save View</chi-link>
             </div>
             <div class="chi-toolbar__end">
-              <button class="chi-button -icon -primary -flat -no-hover" aria-label="Button action">
+              <button class="chi-button -icon -flat" aria-label="Button action">
                 <div class="chi-button__content">
                   <i class="chi-icon icon-x" aria-hidden="true"></i>
                 </div>
               </button>
             </div>
           </div>
         </div>
+        <div class="chi-bulk-actions">
+          <div class="chi-bulk-actions__top">
+            <chi-link href="#">Select all</chi-link>
+            <chi-link href="#">Clear</chi-link>
+          </div>
+          <div class="chi-bulk-actions__start">
+            <div class="chi-bulk-actions__results">
+              <div class="chi-bulk-actions__label">Actions (10 Selected)</div>
+              <div class="chi-form__item">
+                <div class="chi-checkbox">
+                  <input class="chi-checkbox__input" id="checkbox-ba1" type="checkbox" />
+                  <label class="chi-checkbox__label" for="checkbox-ba1">Show Selected Only</label>
+                </div>
+              </div>
+            </div>
+            <div class="chi-bulk-actions__buttons">
+              <div class="chi-bulk-actions__buttons-mobile">
+                <chi-button variant="flat" type="icon" alternative-text="Button action" data-tooltip="Edit">
```

Comment:
Seems like the code snippet is a mix of standard HTML and web components. Thoughts on replacing these with standard HTML?